packages/react-storage/src/components/StorageBrowser/Views/Controls/Table.tsx

```diff
@@ -147,11 +149,24 @@ export const TableControl: TableControl = () => {
           <TableRow key={rowIndex}>
             {columns.map((column) => (
               <TableData key={`${rowIndex}-${column.header}`}>
-                {/* How do we know when a row is supposed to be a button/link? */}
-                {column.key === LOCATION_BUTTON_KEY ? (
-                  <TableDataButton>
-                    <>{row[column.key]}</>
-                  </TableDataButton>
+                {column.key === 'scope' &&
+                (row.type === 'BUCKET' || row.type === 'PREFIX') ? (
+                  <button
+                    key={row['scope']}
+                    onClick={() => {
+                      handleUpdateState({
+                        type: 'ACCESS_LOCATION',
+                        location: {
+                          ...row,
+                          scope: row.scope,
+                          type: row.type,
+                        },
+                      });
+                    }}
+                    type="button"
+                  >
+                    {row.scope}
+                  </button>
```

Comment:
This is not logic tha should be in the `Table` - if we conditionally handle `locations` data here that also means we have to account for conditional handling of location items data and actions data as well. Would using a `renderRowItem` prop be helpful here? Something like:
```
type RenderRowItem = <T>(row: T) => React.JSX.Element;

const SomeControlTable = () => {
  const [data] = useControl({ type:'SOME_DATA' })
  
  const renderRowItem = ({ prefix, type }) => <tr><td>{prefix}</td><td>{type}</>;

  return <Table data={data} renderRowItem={renderRowItem} />
}
```